core/src/main/resources/schema/app/20210611131901_CB-11572_datahub_connected_to_datalake_without_datalakeresource.sql

```diff
@@ -0,0 +1,17 @@
+-- // CB-11572 datahub connected to the datalake without datalakeresource
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE stack ADD COLUMN IF NOT EXISTS datalakecrn varchar(512);
+
+CREATE INDEX idx_stack_datalakecrn ON stack (datalakecrn);
+
+UPDATE stack s
```

Comment:
this update needs to be repeated after this PR is on PROD!

Reply:
this is also in the cb12835